### ISSUE TITLE
Deploying Elastic Endpoint on macOS guide

### ISSUE THREAD

**Issue by caitlinbetz** [elastic/security-docs#3265]
## Description 
We currently have an internal google doc (link [here](https://docs.google.com/document/d/1016kbw02GqqfvxJqg7YCkYZ-fs1upaubplrjHdIRQfA/edit#heading=h.uthyt8e24809))that outlines the steps for deploying elastic defend in maOS environments, including how to deploy using JAMF mdm (page 21). We think it would be helpful to make this guide public and include it in our docs.

**Comment by joepeeples** [comment]
Note to self to check with @nastasha-solomon about this — it sounds like the work she's doing in https://github.com/elastic/security-docs/pull/3161.

**Comment by joepeeples** [comment]
> Note to self to check with @nastasha-solomon about this — it sounds like it overlaps with the work she's doing in #3161.

We discussed and determined that some of the steps in the Google doc were already documented, so this sort of overlaps — but there's still enough unique work here to maintain as separate issues/PRs and for me to continue with this one.

This also intersects with larger efforts we'd like to make to expand Endpoint docs, but OK to proceed with this issue and #3161 and then tie into the larger effort later. Work to do now:

- [ ] Go through the Google doc sections that are already documented and pull out any details that aren't already covered in existing docs.
- [ ] Create new docs for the Jamf/MDM procedures, since those aren't previously documented.
- [ ] Share and Enjoy

**Comment by roxana-gheorghe** [comment]
Updates on Mac deployments are included in this [document](https://docs.google.com/document/d/15J0dRwf2ONv4DBJSkqxHcm1uhvvGTU-Dopnf09PhwlM/edit)

**Comment by natasha-moore-elastic** [comment]
Awaiting clarification on the following:

* Is the [new deployment doc](https://docs.google.com/document/d/15J0dRwf2ONv4DBJSkqxHcm1uhvvGTU-Dopnf09PhwlM/edit#heading=h.twd5rrk83kua) a _replacement_ for the [original doc](https://docs.google.com/document/d/1016kbw02GqqfvxJqg7YCkYZ-fs1upaubplrjHdIRQfA/edit#heading=h.uthyt8e24809), or is it an _addition_ to the original doc? Is there any information from the original doc that we need to keep?
* Internal conversations and unresolved comments in the new doc need to be resolved before we can start drafting up the content.
